yarn.lock

```diff
@@ -5101,10 +5101,10 @@
   resolved "https://registry.yarnpkg.com/@spectrum-css/card/-/card-7.0.2.tgz#4db4772fa6e67ce46a823faca390d57f3afe4512"
   integrity sha512-ftp5BEts14oCFxy9iA0RmndZs0lbbqQCO8Yovd60E73gJmlzlk9SuSiRTGJCGzfPZjm7kZN5q2qfXG9Iwr5Qdw==
 
-"@spectrum-css/checkbox@^8.1.5":
-  version "8.1.5"
-  resolved "https://registry.yarnpkg.com/@spectrum-css/checkbox/-/checkbox-8.1.5.tgz#4dd8ff605f37e8d24b4991528939c2c84ecaba5a"
-  integrity sha512-91xOm+Hj4IrfbPlyNchLf5YW4ohBJlEQq1Urw+mqHSrzjqH9MG+IXLRT9OAY9zhuKTHfEOTqXatgYtfFqR6ILQ==
+"@spectrum-css/checkbox@^14.0.0-next.2":
```

Comment:
Here's where those new Checkbox styles came from. We likely don't want this in this PR?

Reply:
This is needed to create the test case and to surface up the VRTs of downstate.